opm/grid/common/MetisPartition.cpp

```diff
@@ -38,7 +38,7 @@ namespace cpgrid
 {
 
 // We want to use METIS, but if METIS is installed as part of the ScotchMetis package, then the following options are not available.
-#if !(defined(IS_SCOTCH_METIS_HEADER) && IS_SCOTCH_METIS_HEADER)
+#if !(defined(IS_SCOTCH_METIS_HEADER) && IS_SCOTCH_METIS_HEADER > 0)
```

Comment:
This looks a bit to complicated. Isn't the macro always defined?
How about changing FindMETIS a bit instead?
```
diff --git a/cmake/Modules/FindMETIS.cmake b/cmake/Modules/FindMETIS.cmake
index 1f3f623a8..4a7e2758e 100644
--- a/cmake/Modules/FindMETIS.cmake
+++ b/cmake/Modules/FindMETIS.cmake
@@ -35,6 +35,8 @@ if (METIS_INCLUDE_DIRS)
   string(FIND "${metisheader}" "SCOTCH_METIS_PREFIX" IS_SCOTCH_METIS_HEADER)
   if(NOT IS_SCOTCH_METIS_HEADER EQUAL "-1")
     set(IS_SCOTCH_METIS_HEADER 1)
+  else()
+    set(IS_SCOTCH_METIS_HEADER 0)
   endif()
 endif()
```
the we can do a simple 
```
#if IS_SCOTCH_METIS_HEADER
```